### PR TITLE
CY-572 Add option to init and bootstrap with non-default Manager RPM

### PR DESCRIPTION
**Pull request by davidginzbourg**
- Added the `[-u]|[--manager-image-url]` option: URL to the docker manager image. This must be set if you want to download a specific image when running `docl pull-image`
- Added the `[-a]|[--manager-image-commit-sha-url]` option: URL to the `sha1` checksum. This can save time if image already downloaded when the `--manager-image-url` is set.

### CODE REVIEW

File: docl/commands.py [hunk 1]
```diff
@@ -53,14 +53,19 @@
 
 
 @command
+@argh.arg('-u', '--manager-image-url')
+@argh.arg('-m', '--manager-image-docker-tag')
```
Comment (tyacbovi) [inline docl/commands.py]: Is that in use?

File: README.md [hunk 2]
```diff
@@ -66,6 +69,7 @@ To pull the latest built image from S3 (instead of running `bootstrap` and then
 docl pull-image
```
Comment (tyacbovi) [inline README.md]: Add an example for a branch

File: docl/commands.py (hunk 1 above)
Comment (davidginzbourg) [reply docl/commands.py]: Oh yes, that's the way to set the docl image it'll be using (it always used a default value in the config, now I added a param to pass from the CLI)

File: docl/commands.py [hunk 3]
```diff
@@ -53,14 +53,19 @@
 
 
 @command
+@argh.arg('-u', '--manager-image-url')
```
[the remark this answers is not in the document]
Comment (davidginzbourg) [reply docl/commands.py]: Done

File: README.md [hunk 4]
```diff
@@ -20,6 +20,13 @@ Run `docl init` and supply the different configuration options based on your set
 * `--docker-host` should point to the docker endpoint.
 * `--source-root` should point to the root directory in which all cloudify related projects are cloned. This is used for mounting code
   from the host machine to the relevant manager directories.
+* (Optional) `[-u]|[--manager-image-url]` URL to the docker manager image. This must be set if you want to download a specific image when running `docl pull-image`.
+* (Optional) `[-a]|[--manager-image-commit-sha-url]` URL to the `sha1` checksum. This can save time if image already downloaded when 
+the `--manager-image-url` is set. 
+
+Using a dev branch:
```
Comment (tyacbovi) [inline README.md]: It's kinda of repeating it's self with the following, so you can remove it.

Add explanation on the docker flag

**Pull request merged**